core/App/screens/ProofRequest.tsx

```diff
@@ -253,6 +254,7 @@ const ProofRequest: React.FC<ProofRequestProps> = ({ navigation, route }) => {
             }}
           >
             {loading ? <RecordLoading /> : null}
+            {!!proofConnectionLabel && <ConnectionNotification connectionID={proofConnectionLabel} />}
```

Comment:
Since an empty string is considered falsy we don't really need to explicitly convert it to boolean primitive with `!!`.

Reply:
strange, I was getting an error earlier when I didn't have the `!!`. I don't seem to be getting the error now though so maybe I was mistaken